code/modules/religion/religion_sects.dm

```diff
@@ -193,3 +193,148 @@
 	adjust_favor(round(the_cell.charge/3000), L)
 	to_chat(L, "<span class='notice'>You offer [the_cell]'s power to [GLOB.deity], pleasing them.</span>")
 	qdel(I)
+
+
+//Fallen Sect
+
+/datum/religion_sect/fallen
+	name = "Sleeping Gods"
+	desc = "A sect that has fallen into dormancy."
+	convert_opener = "The gods have been silent. Where have you gone?!<br>You have no powers... But, in the darkest hour you will rise to save them all."
+	alignment = ALIGNMENT_GOOD //ancient defenders?
+	altar_icon_state = "sacrificealtar" //just no candles or the like, pretty dull altar
+	/// the current security level as a number, used to give out rewards
+	var/chaos_level = 0
+
+/datum/religion_sect/fallen/on_select()
+	RegisterSignal(SSdcs, COMSIG_GLOB_SECURITYLEVEL_CHANGED, .proc/alert_change)
+
+/datum/religion_sect/fallen/proc/alert_change(new_level)
+	var/increase = chaos_level < new_level ? TRUE : FALSE
+	var/godmessage = ""
+	var/wings = FALSE
+	chaos_level = new_level
+	switch(chaos_level)
+		if(SEC_LEVEL_GREEN)
+			godmessage = "[GLOB.deity] is silent, any murmurs of his power have vanished."
+		if(SEC_LEVEL_BLUE)
+			if(increase)
+				godmessage = "You feel a faint buzz of holy energies for just a moment."
+			else
+				godmessage = "You feel your powers fade as [GLOB.deity] returns to it's slumber..."
+		if(SEC_LEVEL_RED)
+			if(increase)
+				godmessage = "You feel your powers surge as [GLOB.deity] awakens!"
+			else
+				godmessage = "[GLOB.deity] is satisfied with the end of the delta event. The wings are your reward."
+		if(SEC_LEVEL_DELTA)
+			godmessage = "You have been given the full power of [GLOB.deity] to save the station from crisis. [GLOB.deity] watches over you."
```

Comment:
~~my god, this beer nuke is the most terrifying crisis this station has ever seen, send in the angels!~~

Reply:
maybe give each of the chaplains a pinpointer and/or spaceproofing as well, so that they can actually maybe help stop what's causing the delta situation?